pkg/ss/partmgr/rs.go

```diff
@@ -70,22 +70,23 @@ func startNewActors(newParts []*PartCombo, pm *managerImpl) {
 				newp.Run(msg)
 			} else {
 				newp.Run(nil)
-				pm.slog.Info("no trans function", "from", currActorState.String(), "to", currDbState.String())
+				pm.slog.Infow("no trans function", "from", currActorState.String(), "to", currDbState.String())
```

Comment:
.String is expensive. again it should have been a compile warning

Reply:
added logging only for warnings, but printing it as string